x-pack/platform/plugins/shared/streams/server/lib/streams/assets/storage_settings.ts

```diff
@@ -28,6 +29,7 @@ export const assetStorageSettings = {
       [QUERY_KQL_BODY]: types.match_only_text(),
       [QUERY_TITLE]: types.keyword(),
       [QUERY_SEVERITY_SCORE]: types.long(),
+      [QUERY_EVIDENCE]: types.keyword(),
```

Comment:
I'm not sure we want to map this, when would we query by this? (similar for severity but that's already being mapped)